# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: 2f1e8f5b2fc2
# Full Hash: 2f1e8f5b2fc2f5a28fd2f25019108e56de0fbb44
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Rework GMP PluginID plumbing in VideoCondit. r=bwc
#   
#   This prepares VideoConduit for the decoder/encoder factory and its handling of
#   PluginID.
#   
# ==============================================================================

diff -r 777632b3319a -r 2f1e8f5b2fc2 dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Fri Feb 05 10:51:01 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Feb 08 17:23:44 2021 +0100
@@ -223,7 +223,7 @@
 
   virtual void GetRtpSources(nsTArray<dom::RTCRtpSourceEntry>& outSources) = 0;
 
-  virtual uint64_t CodecPluginID() = 0;
+  virtual bool HasCodecPluginID(uint64_t aPluginID) = 0;
 
   virtual void SetPCHandle(const std::string& aPCHandle) = 0;
 